sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/FilterPushdownSuite.scala

```diff
@@ -1459,12 +1470,15 @@ class FilterPushdownSuite extends PlanTest {
   test("SPARK-46707: push down predicate with sequence (without step) through aggregates") {
     val x = testRelation.subquery("x")
 
-    // do not push down when sequence has step param
```

Comment:
maybe just say `Always pushdown sequence as it's deterministic` and remove https://github.com/apache/spark/pull/44975/files#diff-b469906c633fbbcd0d261933b55c09455a94e26ca967b922d1c569c2fcb16415R1483